### PR TITLE
WIP: updating tests for full transaction parsing.

### PR DESCRIPTION
**Pull request by jonored**

### CODE REVIEW

File: tests/lib.sh [hunk 1]
```diff
@@ -36,12 +39,15 @@ check_signature () {
...
+  echo $rfmt > RFMT
+  sfmt="02 $(printf "%x" $slen) ${s_val}"
+  echo $sfmt > SFMT
 
-  SIG="30$(printf "%x" $(($rlen+$slen+4)))$rfmt$sfmt"
+  SIG="30 $(printf "%x" $(($rlen+$slen+4))) $rfmt $sfmt"
+  echo $SIG > sigdump
 
-  xxd -r -ps <<<"$1" | blake2b_lock_hash | xxd -p -r | openssl pkeyutl -verify -pubin -inkey tests/public_key_0_0.pem -sigfile <(xxd -r -ps <<<"$SIG")
+  xxd -r -ps <<<"$1" | tee dumpfile0 | blake2b_p | tee hashfile | xxd -p -r | openssl pkeyutl -verify -pubin -inkey tests/public_key_0_0_0.pem -sigfile <(xxd -r -ps <<<"$SIG")
```
Comment (matthewbauer) [inline tests/lib.sh]: is public_key_0_0_0.pem different from public_key_0_0.pem?

Comment (jonored) [reply tests/lib.sh]: Yes. The 0s are the derivation path after the 44'/309' part.

Comment (matthewbauer) [reply tests/lib.sh]: ok, that file needs to be checked in as well i think

**Comment by jonored** [comment]
Superseded by https://github.com/obsidiansystems/ledger-app-nervos/pull/58

**Pull request closed without merging**